lib/imperative-test.js

```diff
@@ -14,6 +14,8 @@ const kPrepareSubtestOptions = Symbol('prepareSubtestOptions');
 const kRunSubtest = Symbol('runSubtest');
 const kCheckRunSubtest = Symbol('checkRunSubtest');
 
+const nothingSymbol = Symbol('nothing');
```

Comment:
I don't like the fact that you include the word `Symbol` in the identifier name which is similar to Hungarian notation.
Also, maybe use `argumentNotProvided` for both name and description for it to be more self-explanatory?